profile_bluesky/startup/instrument/devices/ad_rigaku_detector.py

```diff
@@ -225,4 +209,4 @@ def xpcs_loop(self, *args, **kwargs):
 logger.info("Sleeping for %s seconds before creating adrigaku object.", _delay)
 ttime.sleep(_delay)
 
-adrigaku = RigakuUfxcDetector(IOC_PREFIX, name="adrigaku")
+adrigaku = RigakuUfxcDetector(IOC_PREFIX, name="adrigaku")
```

Comment:
add a blank line at the end of _every_ Python file

- see: https://stackoverflow.com/a/2287990/1046449
- Following these guidelines will make code easier to review: [PEP 8 -- Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/)